app/flatpak-builtins-ls-remote.c

```diff
@@ -144,7 +144,16 @@ flatpak_builtin_ls_remote (int argc, char **argv, GCancellable *cancellable, GEr
       while (g_hash_table_iter_next (&iter, &key, &value))
         {
           char *ref = key;
-          char *partial_ref = flatpak_make_valid_id_prefix (strchr (ref, '/') + 1);
+          char *partial_ref;
+          const char *slash = strchr (ref, '/');
+
+          if (slash == NULL)
+            {
+              g_debug ("Invalid remote ref %s", ref);
```

Comment:
As you say in the commit message, it's not invalid in the ostree sense. I feel like this message may be unnecessary even though you'd almost never see it.

Reply:
I put the message in to match the identical one a little lower down in the function, which is emitted if the ref can’t be decomposed (i.e. basically under the same circumstances).